### ISSUE TITLE
Problems encountered in repeated experiments: weight decay and log10 index

### ISSUE THREAD

**Issue by whitehb1** [ashutosh1807/PixelFormer#17]
Hello, I encountered some problems when I repeated your experiment: 1. The weight decay of 10−2 mentioned in the paper is not used in the provided code. After adding weight decay=1e-2, the network effect deteriorates a lot. Is weight decay necessary or is the value wrong? 2. On kitti data set, the log10 index in your paper is 0.077, but the log10 of the pre-trained model you provided is only about 0.02~0.03, may I ask if you have encountered this problem?

**Comment by CodeworldKD** [comment]
> Hello, I encountered some problems when I repeated your experiment: 1. The weight decay of 10−2 mentioned in the paper is not used in the provided code. After adding weight decay=1e-2, the network effect deteriorates a lot. Is weight decay necessary or is the value wrong? 2. On kitti data set, the log10 index in your paper is 0.077, but the log10 of the pre-trained model you provided is only about 0.02~0.03, may I ask if you have encountered this problem?

论文中的log10是代码中的log_rms
